ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/account/view/HistoryView.kt

```diff
@@ -551,18 +568,28 @@ private fun HistoryItemExpandedPart(
 ) {
     Column(modifier = modifier) {
         if (transaction.messages.containsValidMemo()) {
+            Text(
+                text = stringResource(id = R.string.account_history_item_message),
```

Comment:
Just a potential follow-up. Although the design doc does not cover this, I think that the iOS side implements this, too. We could leverage plurals, in this case, to cover `Message` vs `Messages:`.